src/InstapaperShareButton.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+import assert from 'assert';
+
+import createShareButton from './utils/createShareButton';
+import objectToGetParams from './utils/objectToGetParams';
+
+function instapaperLink(url, { title }) {
+  assert(url, 'instapaper.url');
+
+  return 'http://www.instapaper.com/hello2' + objectToGetParams({
+    url,
+    text: title,
```

Comment:
Should the name of the parameter be `title`? 

I did not find the official documentation, but https://feedbin.com/help/sharing-read-it-later-services/ and https://gist.github.com/revelt/a70923df337f7923e067 suggest that `url` and `title` would be the parameters to use here.

Reply:
I found [the documentation](https://www.instapaper.com/publishers) and `title` is correct, plus it allows a `description` property. I'll add these.